src/6model/reprs/MVMSpeshCandidate.c

```diff
@@ -0,0 +1,366 @@
+#include "moar.h"
+
+/* This representation's function pointer table. */
+static const MVMREPROps SpeshCandidate_this_repr;
+
+/* Creates a new type object of this representation, and associates it with
+ * the given HOW. */
+static MVMObject * type_object_for(MVMThreadContext *tc, MVMObject *HOW) {
+    MVMSTable *st  = MVM_gc_allocate_stable(tc, &SpeshCandidate_this_repr, HOW);
+
+    MVMROOT(tc, st, {
+        MVMObject *obj = MVM_gc_allocate_type_object(tc, st);
+        MVM_ASSIGN_REF(tc, &(st->header), st->WHAT, obj);
+        st->size = sizeof(MVMSpeshCandidate);
+    });
+
+    return st->WHAT;
+}
+
+/* Initialize the representation. */
+const MVMREPROps * MVMSpeshCandidate_initialize(MVMThreadContext *tc) {
+    return &SpeshCandidate_this_repr;
+}
+
+static void describe_refs(MVMThreadContext *tc, MVMHeapSnapshotState *ss, MVMSTable *st, void *data) {
+    /* TODO */
+}
+
```

Comment:
If not planning to fill this out, it's better not to implement it at all, so the heap snapshot can fall back on the default GC marking approach.